strimzi/datadog_checks/strimzi/metrics.py

```diff
@@ -0,0 +1,103 @@
+    "vertx_pool_completed": "vertx.pool.completed",
+    "vertx_pool_in_use": "vertx.pool.in_use",
+    "vertx_pool_queue_pending": "vertx.pool.queue_pending",
+    "vertx_pool_queue_time_seconds": "vertx.pool.queue_time_seconds",
+    "vertx_pool_queue_time_seconds_max": "vertx.pool.queue_time_seconds.max",
+    "vertx_pool_ratio": "vertx.pool.ratio",
+    "vertx_pool_usage_seconds": "vertx.pool.usage_seconds",
+    "vertx_pool_usage_seconds_max": "vertx.pool.usage_seconds.max",
+}
+
+COMMON_METRICS_MAP = {
+    "jvm_buffer_count_buffers": "jvm.buffer.count_buffers",
+    "jvm_buffer_memory_used_bytes": "jvm.buffer.memory_used_bytes",
+    "jvm_buffer_total_capacity_bytes": "jvm.buffer.total_capacity_bytes",
+    "jvm_classes_loaded_classes": "jvm.classes.loaded_classes",
+    "jvm_classes_unloaded_classes": "jvm.classes.unloaded_classes",
+    "jvm_gc_live_data_size_bytes": "jvm.gc.live_data_size_bytes",
+    "jvm_gc_max_data_size_bytes": "jvm.gc.max_data_size_bytes",
+    "jvm_gc_memory_allocated_bytes": "jvm.gc.memory_allocated_bytes",
+    "jvm_gc_memory_promoted_bytes": "jvm.gc.memory_promoted_bytes",
+    "jvm_gc_pause_seconds": "jvm.gc.pause_seconds",
+    "jvm_gc_pause_seconds_max": "jvm.gc.pause_seconds.max",
+    "jvm_memory_committed_bytes": "jvm.memory.committed_bytes",
+    "jvm_memory_max_bytes": "jvm.memory.max_bytes",
+    "jvm_memory_used_bytes": "jvm.memory.used_bytes",
+    "jvm_threads_daemon_threads": "jvm.threads.daemon_threads",
+    "jvm_threads_live_threads": "jvm.threads.live_threads",
+    "jvm_threads_peak_threads": "jvm.threads.peak_threads",
+    "jvm_threads_states_threads": "jvm.threads.states_threads",
+    "process_cpu_usage": "process.cpu_usage",
+    "strimzi_reconciliations": "reconciliations",
+    "strimzi_reconciliations_duration_seconds": "reconciliations.duration_seconds",
+    "strimzi_reconciliations_duration_seconds_max": "reconciliations.duration_seconds.max",
+    "strimzi_reconciliations_failed": "reconciliations.failed",
+    "strimzi_reconciliations_locked": "reconciliations.locked",
+    "strimzi_reconciliations_periodical": "reconciliations.periodical",
+    "strimzi_reconciliations_successful": "reconciliations.successful",
+    "strimzi_resource_state": "resource.state",
+    "strimzi_resources": "resources",
+    "strimzi_resources_paused": "resources.paused",
+    "system_cpu_count": gauge("system.cpu_count"),
+    "system_cpu_usage": "system.cpu_usage",
+    "system_load_average_1m": "system.load_average_1m",
+}
+
+CLUSTER_OPERATOR_METRICS_MAP = {
+    "strimzi_reconciliations_already_enqueued": "reconciliations.already_enqueued",
+}
+CLUSTER_OPERATOR_METRICS_MAP.update(COMMON_METRICS_MAP)
+CLUSTER_OPERATOR_METRICS_MAP.update(VERTX_METRICS_MAP)
+
+TOPIC_OPERATOR_METRICS_MAP = copy.deepcopy(COMMON_METRICS_MAP)
+TOPIC_OPERATOR_METRICS_MAP.update(VERTX_METRICS_MAP)
+
+USER_OPERATOR_METRICS_MAP = copy.deepcopy(COMMON_METRICS_MAP)
+USER_OPERATOR_METRICS_MAP.update(COMMON_METRICS_MAP)
```

Comment:
Should this be:
```suggestion
USER_OPERATOR_METRICS_MAP.update(VERTX_METRICS_MAP)
```

Reply:
ah thee's no VERTX metrics for this endpoint, good catch, it's a copy/paste issue